openslides/agenda/signals.py

```diff
@@ -15,65 +15,50 @@ def validate_start_time(value):
     try:
         datetime.strptime(value, '%d.%m.%Y %H:%M')
     except ValueError:
-        raise ValidationError(_('Invalid input.'))
+        raise DjangoValidationError(_('Invalid input.'))
 
 
-# TODO: Reinsert the datepicker scripts in the template
-
 def setup_agenda_config(sender, **kwargs):
     """
     Receiver function to setup all agenda config variables. It is connected to
     the signal openslides.config.signals.config_signal during app loading.
     """
-    # TODO: Insert validator for the format or use other field carefully.
     agenda_start_event_date_time = ConfigVariable(
         name='agenda_start_event_date_time',
         default_value='',
-        form_field=forms.CharField(
-            validators=[validate_start_time, ],
-            widget=forms.DateTimeInput(format='%d.%m.%Y %H:%M'),
-            required=False,
-            label=ugettext_lazy('Begin of event'),
-            help_text=ugettext_lazy('Input format: DD.MM.YYYY HH:MM')))
+        html_input_type='text',  # TODO: Use 'datetime' here. Use a better validator.
```

Comment:
Hm. Ok. But then the question is whether we should use another mapping: string instead of text? boolean instead of checkbox? integer instead of number? What do you think?